pkgs/applications/networking/instant-messengers/patchwork/default.nix

```diff
@@ -0,0 +1,70 @@
+{ stdenv, writeScriptBin, path, fetchurl, fetchFromGitHub
+, nodejs-8_x, yarn, python, electron }:
+
+with builtins;
+
+let
+  version = "3.8.10";
+
+  nodejs = nodejs-8_x;
+
+  yarn2nix = (import (fetchFromGitHub {
+    owner = "moretea";
+    repo = "yarn2nix";
+    rev = "d6e05a521bd92b2647bb7e853363d234f21b2cfd";
+    sha256 = "1nvpii9p41vrb6zvr8rqcvmycrl6lnzzaif85qj1aavizncgb4wy";
+  }) {
+    inherit nodejs;
+    pkgs = import path {};
+    yarn = yarn.override { inherit nodejs; };
+  });
+
+  nodeHeaders = fetchurl {
+    url = "https://nodejs.org/download/release/v${nodejs.version}/node-v${nodejs.version}-headers.tar.gz";
+    sha256 = "1993kcghzr56zmw5sdj8wr8c42mna25806bcjknfxnh62zl4hwpg";
+  };
```

Comment:
I'm doing the exact same thing in another project and it breaks every time I update nixpkgs and the nodejs has been bumped. I think the nodejs package should export the headers in another output like `nodejs.dev`